src/extensions/volumes-and-segmentations/entry-root.ts

```diff
@@ -151,6 +151,9 @@ export class VolsegEntryData extends PluginBehavior.WithSubscribers<VolsegEntryP
             // do nothing
         }
 
+        const volumeVisual = this.findNodesByTags(VOLUME_VISUAL_TAG)[0];
+        if (volumeVisual) this.currentVolume.next(volumeVisual.transform);
```

Comment:
This isn't correct if there is more than 1 segmentations loaded.

Need to use the `StateSelection.findAncestor(this.plugin.state.data.tree, this.plugin.state.data.cells, e.ref, a => a.transform.ref === ref)` as well here.

Reply:
this.findNodesByTag only searches the subtree of current entry root.
So it should be OK.